_sass/_navigation-header.scss

```diff
@@ -358,31 +360,31 @@ $max-container: 1440px;
                                     margin-right: 18px;
                                     &:hover,
                                     &.in-category {
-                                        color: $primary-open-sky-s2;
+                                        color: var(--nav-link-hover-color);
                                         text-decoration: underline;
-                                        text-decoration-color: $primary-open-sky-s2;
+                                        text-decoration-color: var(--nav-link-hover-color);
                                         text-decoration-thickness: 2px;
                                         text-underline-offset: 6px;
                                     }
                                 }
                                 @media screen and (max-width: $header-banner-mobile-breakpoint) {
                                     @include body-text;
-                                    color: $primary-open-sky-t3;
+                                    color: #FFFFFF;
```

Comment:
This will be an issue in light mode right?

Reply:
This is the default color that's overridden later but I've changed it to use the variable - thanks.